pactsdk/constant_product_calculator.py

```diff
@@ -44,6 +47,59 @@ def get_constant_product_minted_liquidity_tokens(
     return lt_b if lt_a > lt_b else lt_a
 
 
+def get_constant_product_zap_params(
+    liq_a: int,
+    liq_b: int,
+    zap_amount: int,
+    fee_bps: int,
+    pact_fee_bps: int,
+) -> "ZapParams":
+    from .zap import ZapParams
```

Comment:
I would move this to `zap.py` to avoid the circular dependency issue.
The same for js sdk.